src/api/PDFPage.ts

```diff
@@ -892,45 +889,52 @@ export default class PDFPage {
     assertOrUndefined(options.maxWidth, 'options.maxWidth', ['number']);
     assertOrUndefined(options.wordBreaks, 'options.wordBreaks', [Array]);
     assertIsOneOfOrUndefined(options.blendMode, 'options.blendMode', BlendMode);
+    assertIsOneOfOrUndefined(options.textAlignment, 'options.allign', TextAlignment);
```

Comment:
typo in `options.allign`, should be `.align`?

Reply:
thanks for the remark, this is now fixed in the latest version of the code